core/tee/tee_svc_storage.c

```diff
@@ -1069,8 +1097,12 @@ TEE_Result syscall_storage_obj_write(unsigned long obj, void *data, size_t len)
 		goto exit;
 
 	o->info.dataPosition += len;
```

Comment:
But the write operation have overwrote the data, the data position actually have changed? So we ignore this overwrite?